test/point/distance_to_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'minitest/autorun'
+require 'geometry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.